src/metrics/metric/id/m3/id_bench_test.go

```diff
@@ -0,0 +1,43 @@
+package m3
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var benchmarkIsRollupIDResult bool
```

Comment:
i'm guessing this is to validate some assumption around escaping/inlining. can you add a comment + test which does the assertion

Reply:
Good call! Added a test which literally checks that allocs are zero--not super usual but also not a bad idea here.